charts/cass-operator-chart/values.yaml

```diff
@@ -9,5 +9,5 @@ webhookClusterRoleName: cass-operator-webhook
 webhookClusterRoleBindingName: cass-operator-webhook
 deploymentName: cass-operator
 deploymentReplicas: 1
-image: "datastax/cass-operator:1.4.0"
+defaultImage: "datastax/cass-operator:1.4.0"
```

Comment:
do we have to change this key?

Reply:
also you should get this to 1.4.1 :)

Reply:
This has actually been in draft and I just got around to merging in master again. It is using the correct image.

It is important for me to be able to distinguish the user specifying an operator image explicitly vs not specifying one. That is reason the I changed `image` -> `defaultImage`. The user will still use `--set image=whatever` to set the operator image.

The general principal here is to respect the user's most specific and explicit instruction for where to get the image. So if the user only specifies the registry, but not the operator image, then we will get the operator image from the registry they gave. If the user gives us both a registry _and_ an image, then we will not modify the image location that they gave us explicitly for the operator. Really, we could just not have the image mentioned at all in the values.yaml, and just have this literal in the deployment.yaml, but I thought it would be easier to find it in this file to update.

Reply:
I think this block from the deployment.yaml will make it clearer:

```
      - name: cass-operator
        {{- if .Values.image }}
        image: {{ .Values.image }}
        {{- else if .Values.registryName }}
        image: {{ printf "%s/%s" .Values.registryName .Values.defaultImage }}
        {{- else }}
        image: {{ .Values.defaultImage }}
        {{- end }}
```

Maybe there is a better way to do that though?